src/Morphic-Core/MiniWorld.class.st

```diff
@@ -0,0 +1,502 @@
+"
+I represent a litlle world for an other window open out of the pharo window
+
+I create a new world and i know my render, my state and my root.
+
+    Instance Variables
+	root:		MorphicRoot
+	state:		WorldState
+	submorph:		morphe
+	worldRendered:		worldrendered
```

Comment:
Maybe it could be good to add a one line comment to explain what those are. 

For example I guess that the Morphic root is the class that manages the different mini world but I'm not sure. What is the submorph? The morph encapsulating all the morphs of this miniworld? If yes, why isn't it the MiniWorld itself?